src/js/controller/tracks-loader.ts

```diff
@@ -96,8 +110,11 @@ function xhrSuccess(xhr, track, successHandler, errorHandler) {
     }
 }
 
-function loadVttParser() {
-    return require.ensure(['parsers/captions/vttparser'], function (require) {
-        return require('parsers/captions/vttparser').default;
-    }, chunkLoadWarningHandler(301131), 'vttparser');
+function loadVttParser(): any {
+    return require.ensure(['parsers/captions/vttparser'], 
+        function (require: NodeRequire): Promise<VTTParser> {
+            return require('parsers/captions/vttparser').default;
+        }, 
+        chunkLoadWarningHandler(301131), 'vttparser'
+    );
```

Comment:
you tried?
```suggestion
    ) as Promise<VTTParser> ;
```

Reply:
Yep. Ended up reverting to `any` since there were issues with every trick I've tried to type the return. So far the only thing that works is to cast to unknown before casting to Promise<VTTParser>, which doesn't seem to have an advantage over `any`.